lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -334,6 +367,13 @@ def createMetadataSection(self):
             self.defineCustom(metaDataControl)
             self.defineCustom(usdNoticeControl)
 
+    def createCustomExtraAttrs(self):
+        # We are not using the maya default "Extra Attributes" section
+        # because we are using custom widget for array type and it's not
+        # possible to inject our widget inside the maya "Extra Attributes" section.
+        extraAttrs = [attr for attr in self.attrS.attributeNames if attr not in self.addedAttrs]
```

Comment:
Do we also need to ignore any attributes in self.suppressedAttrs? It is possible that someone can call suppress() on any attribute and we don't want it ending up in extra attributes.

Reply:
Suppressing attribute will not be an issue as long as we are using the suppress function from the template. The suppress function will add the attribute to suppress into self.suppressedAttrs array which is used by addControls(). So extraAttrs will contain suppressed attributes but they will be ignored.

Reply:
Ah right. Can you add a comment here explaining that.

Reply:
done